### PR TITLE
Support german language

### PR DESCRIPTION
**Pull request by Felix10R**

### CODE REVIEW

File: res/lang/de.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    "week_graph":         "Wochengraph",
+    "turnips_patterns": {
+        "fluctuating":    "Schwankend",
+        "large_spike":    "Große Spitze",
+        "decreasing":     "Abnehmend",
+        "small_spike":    "Kleine Spitze"
+    },
+
+    "npcs": {
+        "none":           "keiner",
```
Comment (averne) [inline res/lang/de.json:39]:
```suggestion
        "none":           "Keiner",
```
Capitalization for consistency with other languages

File: res/lang/de.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        "flick":          "Carlson",
+        "kicks":          "Schubert",
+        "leif":           "Gerd",
+        "redd":           "Reiner",
+        "gullivarrr":     "Gullivarrr"
+    },
+
+    "hemisphere":         "Hemisphäre: %s",
+    "weather_seed":       "Wetter-Seed: %d (%#x)",
+    "weather_url_tip":    "Trage diesen Seed auf wuffs.org/acnh/weather ein, &\num das Wetter und Meteorschauer vorherzusagen",
```
Comment (averne) [inline res/lang/de.json:55]:
```suggestion
    "weather_url_tip":    "Trage diesen Seed auf wuffs.org/acnh/weather ein, & um das Wetter\nund Meteorschauer vorherzusagen",
```
Move the newline a little further so as to use more horizontal space

**Comment by averne** [comment]
Thanks for the PR, see nitpicks

**Comment by Felix10R** [comment]
Your changes look good, but the symbol "&" was accidentally added by me (I'm sorry :D). After removing, the changes can be merged. And I just noticed that Celeste is missing in every translation. In German, she is called "Eufemia".

File: res/lang/de.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        "decreasing":     "Abnehmend",
+        "small_spike":    "Kleine Spitze"
+    },
+
+    "npcs": {
+        "none":           "keiner",
+        "gulliver":       "Gulliver",
+        "label":          "Minna",
+        "saharah":        "Aziza",
+        "wisp":           "Buhu",
```
Comment (averne) [inline res/lang/de.json:43]:
```suggestion
        "wisp":           "Buhu",
        "celeste":        "Eufemia", 
```

**Comment by averne** [comment]
LGTM, I've also fixed the missing Celeste.
Thanks for the PR and sorry for taking a few days, I was busy with other stuff.

**Pull request merged**